README.md

```diff
@@ -1,5 +1,62 @@
 # BC Parks Foundation
 
+BC Parks Foundation is a Mobile App build with React Native Technology that can be deployed in both ios and Android (with some ajustments) platforms.
+
+## <img src="./client/js/assets/images/discoverWhite.png" >
+
+## Some Screenshots from Mobile App:
```

Comment:
Please add only one gif showing some of your app's screens.

Reply:
We'll do when we finalize the app